garglk/garglk.ini

```diff
@@ -42,7 +42,13 @@ caps          0               # Force uppercase input  -- 0=off 1=on
 graphics      1               # enable graphics
 sound         1               # enable sound
 
-fullscreen    0               # set to 1 for fullscreen (Windows/Unix only)
+fullscreen    0               # set to 1 for fullscreen
+
+hires         1               # set to 1 to enable Retina display (macOS) / HiDPI (Windows)
+zoom          1.0             # set display zoom
+
+hires         1               # set to 1 to enable Retina display (macOS) / HiDPI (Windows)
+zoom          1.0             # set display zoom
```

Comment:
These config options are listed twice.

Since the Windows bit has been removed, should this now just refer to macOS?

Qt has support for hidpi (https://doc.qt.io/qt-5/highdpi.html) but I don't have any clue how it works with Gargoyle on Windows (or Unix, for that matter), since I don't have a hidpi display.  It'd be nice to get tested on Windows/Unix and update this, but I'm also OK with a Mac-only fix for the time being, given that this is a known issue on Mac, and I've not seen a single bug report regarding Windows and hidpi.

Reply:
In Windows 10 you can set any screen to have a higher pixel density, by changing the display scale. It gets reported through to web browsers, so I assume Qt will identify the pixel density too.